### PR TITLE
Intersection Point in Y shaped Linked List

### PR DESCRIPTION
**Pull request by Rishikeshrajrxl**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(I have closed the previous pull request due to messed up, and here I have created new PR.)

### Description

(Here I have Done two assigned work)
1) Intersection Point in Y shaped Linked List.
2) Created a Contributor's Guideline 

### Checklist

- [X] I've read the contribution guidelines.
- [X] I've checked the issue list before deciding what to submit.
- [X] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

( #495 )
(#542 )

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@Rishikeshrajrxl add examples, time and space complexity of the program. Move this code to `cp` section, not in `ds` section. Also, remove unnecessary files in the commit.

File: C-Plus-Plus/ds/Intersection Point of Y shaped LinkedList.cpp
Comment (iamrajiv) [inline C-Plus-Plus/ds/Intersection Point of Y shaped LinkedList.cpp:32]: @Rishikeshrajrxl avoid using this type of lines and patterns across the program.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Comment by Rishikeshrajrxl** [comment]
> @Rishikeshrajrxl add examples, time and space complexity of the program. Move this code to `cp` section, not in `ds` section. Also, remove unnecessary files in the commit.

@iamrajiv , There is no unnecessary file. There are some files that contain Images for the Contribution Page. Please have a look over there.

File: C-Plus-Plus/ds/Intersection Point of Y shaped LinkedList.cpp
Comment (Rishikeshrajrxl) [reply C-Plus-Plus/ds/Intersection Point of Y shaped LinkedList.cpp:32]: > @Rishikeshrajrxl avoid using this type of lines and patterns across the program.

I have made this program User interactive which has a good Interface. And these Lines are to separate the function individually.
Previously @plazzy99  appreciated the same program and my work.

Comment (Rishikeshrajrxl) [reply C-Plus-Plus/ds/Intersection Point of Y shaped LinkedList.cpp:32]: These lines are to separate the functions inside the program, if I remove this the interface will look messed.

**Review by vatsalkesarwani12 — CHANGES_REQUESTED** [review]
@Rishikeshrajrxl please make one pull request for one issue

**Comment by Rishikeshrajrxl** [comment]
> @Rishikeshrajrxl please make one pull request for one issue

ok, from next time I will take care of this, Now for now, please review it merge it somehow, if possible.

**Comment by vatsalkesarwani12** [comment]
sorry, we cannot merge this pull request you have to make changes before this request to be merged

**Pull request closed without merging**
